app/auth/views/activate.py

```diff
@@ -64,4 +64,4 @@ def activate():
         return redirect(next_url)
     else:
         LOG.d("redirect user to dashboard")
-        return redirect(url_for("dashboard.index"))
+        return redirect(url_for("onboarding.account_activated"))
```

Comment:
can we serve the account activated page directly without a redirection?

Reply:
It's easier to debug if they are separate. We don't need to do the full flow to test each browser.